packages/manager/apps/web/client/app/hosting/cdn/shared/leave-confirm-settings/confirm/hosting-shared-confirm-settings.routing.js

```diff
@@ -0,0 +1,22 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.hosting.cdn.shared.confirmSettings', {
+    url: '/add-cache-rule',
```

Comment:
What's happen if we reach this url directly ?

Reply:
if required params are missed it will redirect

Reply:
we cant because i implemented a rule to avoid that.

Reply:
no we can, i implemented a rule to avoid that.